runtime/trappist/src/lib.rs

```diff
@@ -324,7 +327,7 @@ impl pallet_collator_selection::Config for Runtime {
 	type UpdateOrigin = CollatorSelectionUpdateOrigin;
 	type PotId = PotId;
 	type MaxCandidates = ConstU32<1000>;
-	type MinCandidates = ConstU32<5>;
+	type MinCandidates = ConstU32<0>;
```

Comment:
are we downgrading this so we don't allow any collators to join? that would make sense.

Reply:
See referenced PR on the Contracts parachain.